api-tester/src/main/java/com/revenuecat/apitester/java/TransactionAPI.java

```diff
@@ -8,9 +8,7 @@
 final class TransactionAPI {
     static void check(final Transaction transaction) {
         final String transactionIdentifier = transaction.getTransactionIdentifier();
-        final String revenuecatId = transaction.getRevenuecatId();
         final String productIdentifier = transaction.getProductIdentifier();
-        final String productId = transaction.getProductId();
```

Comment:
I left some thoughts over slack, I think we should strive to keep anything that isn't an absolute PITA to keep around. 

Being overly aggressive with removing deprecated code means disrupting developers, and it will result in unnecessary friction as well as less people upgrading. And the last major version of our SDK was released this year, I bet a lot of people will be upgrading 2 major versions at a time